### PR TITLE
[ minor ] bugfix in printing error log (model_commo_properties / LossScale)

### PR DESCRIPTION
**Pull request by EunjuYang**
- This commit fixes a minor error to print error log.
- This commit adds a condition to print the error log.

Relevant issue: #2826 

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]
LGTM, but in this case, how about comparing float with stricter standard? 

something like:
```C++
if (std::fpclassify(value) == FP_ZERO)
...
```

**Review by djeong20 — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Review by myungjoo — CHANGES_REQUESTED** [review]
Check the return.

File: nntrainer/models/model_common_properties.cpp [hunk 1]
```diff
@@ -43,7 +43,8 @@ ModelTensorDataType::ModelTensorDataType(ModelTensorDataTypeInfo::Enum value) {
 LossScale::LossScale(float value) { set(value); }
 
 bool LossScale::isValid(const float &value) const {
-  ml_loge("Loss scale cannot be 0");
+  if (std::fpclassify(value) == FP_ZERO)
+    ml_loge("Loss scale cannot be 0");
```
[the remark this answers is not in the document]
Comment (EunjuYang) [reply nntrainer/models/model_common_properties.cpp:48]: I missed it! Thank you for pointing it out. I updated the part. Thank you.

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**